services/director-v2/src/simcore_service_director_v2/modules/dynamic_sidecar/scheduler/_core/_events_utils.py

```
@@ -440,3 +446,18 @@ async def prepare_services_environment(
     )
 
     scheduler_data.dynamic_sidecar.is_service_environment_ready = True
+
+
+async def get_hardware_info(
+    app: FastAPI, scheduler_data: SchedulerData
+) -> HardwareInfo | None:
+    hardware_info: HardwareInfo | None = None
+    if scheduler_data.wallet_info and scheduler_data.pricing_info:
+        resource_tracker_pricing_units_repo = get_repository(
+            app, ResourceTrackerPricingUnitsRepository
+        )
+        with suppress(ResourceTrackerPricingUnitsError):
```

Q: Why do you want to suppress the error?